.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,47 @@
+name: Bug Report
+description: Report a bug or unexpected behaviour with the website.
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking your time to report this bug!
+
+  - type: textarea
+    id: how-to-reproduce
+    attributes:
+      label: Reproduction steps
+      description: "Please describe how to reproduce the bug."
+    validations:
+      required: true
+
+  - type: textarea
+    id: expected-results
+    attributes:
+      label: Expected results
+      description: "What should happen when you follow the reproduction steps?"
+    validations:
+      required: true
+
+  - type: textarea
+    id: actual-results
+    attributes:
+      label: Actual results
+      description: "What actually happened?"
+    validations:
+      required: true
+
+  - type: input
+    id: device-and-os
+    attributes:
+        label: What device are you using and what OS is it running?
+    validations:
+        required: true
+
+  - type: input
+    id: browser
+    attributes:
+      label: Browser information.
+      description: "The browser you're using and its version, if you know it."
```

```suggestion
      description: "The browser you were using and its version, if you know it."
```
It occurs to me that if we ask the reporter which browser they *are* using, they may input the browser that they are using to report the issue, which might be different from the one the issue occurs in (if they're reporting a mobile bug on their desktop, for example).